### PR TITLE
Update bucket system

### PR DESCRIPTION
**Pull request by Palakp41**
Closes #

Describe your proposed changes here.
- Update bucket list to hide rename and delete buttons for system buckets
- Filter out system buckets from the list when checking if limits have been exceeded

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

Screen Shot 2019-09-09 at 10 27 23 AM

### CODE REVIEW

File: ui/src/buckets/components/BucketCard.tsx [hunk 1]
```diff
@@ -80,6 +75,21 @@ class BucketRow extends PureComponent<Props & WithRouterProps> {
     )
   }
 
+  private get renameButton() {
+    const {bucket} = this.props
+    if (bucket.type === 'user') {
+      console.log('here')
```
Comment (chnn) [inline ui/src/buckets/components/BucketCard.tsx]:
```suggestion
```

File: ui/src/buckets/components/BucketCard.tsx [hunk 2]
```diff
@@ -80,6 +75,21 @@ class BucketRow extends PureComponent<Props & WithRouterProps> {
...
+      return (
+        <Button
+          text="Rename"
+          testID="bucket-rename"
+          size={ComponentSize.ExtraSmall}
+          onClick={this.handleRenameBucket}
+        />
+      )
+    } else return <></>
```
Comment (chnn) [inline ui/src/buckets/components/BucketCard.tsx]:
```suggestion
    } else {
      return null
    }
```

File: ui/src/buckets/components/BucketContextMenu.tsx [hunk 3]
```diff
@@ -33,23 +31,32 @@ export default class BucketContextMenu extends PureComponent<Props> {
...
+        >
+          <Context.Item
+            label="Confirm"
+            action={onDeleteBucket}
+            value={bucket}
+            testID="context-delete-task"
+          />
+        </Context.Menu>
+      )
+    } else return <></>
```
Comment (chnn) [inline ui/src/buckets/components/BucketContextMenu.tsx]:
```suggestion
    } else {
      return null
    }
```

File: ui/src/cloud/actions/limits.ts [hunk 4]
```diff
@@ -292,7 +292,10 @@ export const checkBucketLimits = () => async (
     } = getState()
 
     const bucketsMax = extractBucketMax(limits)
-    const bucketsCount = list.length
+    const buckets = list.filter(function(bucket) {
```
Comment (chnn) [inline ui/src/cloud/actions/limits.ts]:
```suggestion
    const buckets = list.filter(bucket => {
```

File: ui/src/buckets/components/BucketCard.tsx (hunk 1 above)
Comment (Palakp41) [reply ui/src/buckets/components/BucketCard.tsx]: Removed it :)

**Pull request merged**